miner/miner.go

```diff
@@ -102,6 +102,19 @@ var DefaultConfig = Config{
 	NewPayloadTimeout: 2 * time.Second,
 }
 
+// DefaultOpBNBConfig contains default opBNB settings for miner.
+var DefaultOpBNBConfig = Config{
+	GasCeil:  150000000,
+	GasPrice: big.NewInt(params.GWei),
+
+	// The default recommit time is chosen as two seconds since
+	// consensus-layer usually will wait a half slot of time(6s)
+	// for payload generation. It should be enough for Geth to
+	// run 3 rounds.
+	Recommit:          2 * time.Second,
```

Comment:
seems the context does not match opbnb

Reply:
This is based on the previous startup document. In fact, this configuration is useless. This is the previous block production configuration of Ethereum. For us, we only have the corresponding parameters for the master sequencer to produce blocks and control the block production by other conditions. 
Remove it. Please help confirm it again.